### DISCUSSION TITLE
Filtering on Datapane

### DISCUSSION THREAD

**Discussion by rizkysifaul** [datapane/datapane#99] [Help]
Hi, I want to ask, can we create filtering on the Datapane? For example, we have data with many columns, at the same point we want to show one plot, but using this filter I can show another plot, so it depends on what people are choosing on the filter. Thanks.

**Comment by lanthias** [comment]
I have been looking into this a bit, and it _should_ be possible with Altair and maybe Plotly, though it's relatively tricky considering that Datapane reports don't have are running Python server.

What viz library were you hoping to use?

**Reply by rizkysifaul** [reply]
I mean it should be done with the helps of visualization packages such as Altair and Plotly, currently, I am much more using Plotly Express and Folium, maybe Datapane can create some API that connected between selecting filter with the viz
